agent/hcp/telemetry/otel_sink.go

```diff
@@ -75,9 +91,12 @@ type OTELSink struct {
 // NewOTELReader returns a configured OTEL PeriodicReader to export metrics every X seconds.
 // It configures the reader with a custom OTELExporter with a MetricsClient to transform and export
 // metrics in OTLP format to an external url.
-func NewOTELReader(client MetricsClient, endpointProvider EndpointProvider, exportInterval time.Duration) otelsdk.Reader {
-	exporter := NewOTELExporter(client, endpointProvider)
-	return otelsdk.NewPeriodicReader(exporter, otelsdk.WithInterval(exportInterval))
+func NewOTELReader(client MetricsClient, endpointProvider EndpointProvider) otelsdk.Reader {
```

Comment:
Should we keep the parameterization and still pass it in from the other file to make this extensible with the exported default to avoid making it HCP use case specific?

Reply:
I thought about that but right now it's only created in one place. My preference would be to add that when there's another consumer that might set different values. But lmk if you feel strongly about this remaining parameterized for easier use by other users down the line

Reply:
yeah I don't mind we can easily change that in the future